test/test_helper.rb

```diff
@@ -0,0 +1,20 @@
+# require 'simplecov'
+# SimpleCov.start
+# WebMock.disable_net_connect!(allow: %w{codeclimate.com})
+
+ENV['RACK_ENV'] = 'test'
+require 'minitest/autorun'
+require 'webmock/minitest'
+require 'rack/test'
+
+require File.expand_path '../../app.rb', __FILE__
+
+def load_json_fixture(file)
+   root = File.dirname(File.absolute_path(__FILE__))
+   JSON.parse(File.read(File.join(root, 'fixtures', "#{file.to_s}.json")))
```

Comment:
I have previously observed memory leaks with `JSON.parse` unless you pass it a `String`... so, not a `File.read` which should be the same thing, but somehow in runtime is not. Shrug.

Reply:
Probably doesn't matter with unit tests though.